docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_circularindexscrollbar.htm

```diff
@@ -14,6 +14,7 @@ <h1>Circular Index Scroll Bar</h1>
 <p>If you spin the rotary quickly, an indicator will be displayed and you can move in an index unit.</p>
 <p>The indicator will disappear if no rotary action is detected for a second.</p>
 <p>This component fires a select event when the index characters are selected.</p>
+<img src="../../images/Circularindexscrollbar.png" alt="photo of circleindex element in Tizen">
```

Comment:
Maybe a line break is needed here.

The "Use example" button is right next to the image.

![image](https://user-images.githubusercontent.com/9958374/71229718-081df100-232a-11ea-85b7-03eeb4e4d636.png)

Reply:
Done